http-generator-core/src/main/java/io/avaje/http/generator/core/ElementReader.java

```diff
@@ -41,13 +46,13 @@ public class ElementReader {
     this.element = element;
     this.type = type;
     this.rawType = rawType;
-    this.shortType = Util.shortName(rawType);
-    this.contextType = ctx.platform().isContextType(rawType);
-    this.typeHandler = TypeMap.get(rawType);
+    shortType = Util.shortName(rawType);
```

Comment:
Noting that style wise I prefer using `this.` in constructors as it has no cost, makes the indentation uniform and emphasizes that the assignment is being done on construction.

Wondering why are preferring to remove the `this.`?

Reply:
my IDE auto takes it out for some reason